modules/library/main/src/main/java/org/geotools/geometry/jts/CircularString.java

```diff
@@ -212,7 +212,11 @@ public String getGeometryType() {
         return "CircularString";
     }
 
-    protected Geometry reverseInternal() {
+    public CircularString reverse() {
+        return (CircularString) super.reverse();
+    }
+
+    public CircularString reverseInternal() {
```

Comment:
Do we want to raise the access here? Internal feels like it shouldn't be public.

Reply:
It should not be, mistake made upstream in JTS (sorry)